src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -2335,6 +2334,51 @@ public void EndToEndWarnAsErrors()
             success.ShouldBeFalse();
         }
 
+        [Trait("Category", "netcore-osx-failing")]
+        [Trait("Category", "netcore-linux-failing")]
```

Comment:
Were you able to figure out why this is hanging? IIRC I reproed on my Mac and then forgot to ask if you wanted to look at it :(

Reply:
I have have reproduced it on WSL. Got and analyzed its hanging process dump, however that hanging process was entry process blocked waiting on read from pipe - i.e. wrong process. What was weird was that out of node process wasn't there. It most probably crashed otherwise there would be error logged by entry node. What's even more weird is that when I started the same command from shell by `export MSBUILDNOINPROCNODE=1 && /home/rokonecn/dev/msbuild/.dotnet/dotnet /home/rokonecn/dev/msbuild/artifacts/bin/Microsoft.Build.CommandLine.UnitTests/Debug/net5.0/MSBuild.dll /tmp/umsex53t.mvo/Temporary258ec66ef0b94800b53904586527acd9/TestSolution.sln /p:Configuration=Debug /t:Build` it worked.
Given my limited experience with linux unit test debugging, I've considered it too time consuming and moved on.
Does MSBUILDDEBUGONSTART works on linux? Is there a way to debug it on linux somehow? Is there msbuild crash log somewhere on linux?